.github/renovate.json

```diff
@@ -20,6 +20,30 @@
       "matchDepTypes": ["dependencies", "devDependencies"],
       "enabled": true
     },
+    {
+      "groupName": "npm dependencies bump",
+      "matchManagers": ["npm"],
+      "rangeStrategy": "update-lockfile",
+      "matchDepTypes": ["dependencies", "devDependencies"],
+      "matchPackagePatterns":[
+        "^@microsoft/fast",
```

Comment:
Actually maybe we should just exclude our manually managed / pinned packages and let everything else update. I'll switch to that.